apps/src/lib/kits/maker/boards/microBit/MicroBitBoard.js

```diff
@@ -142,9 +163,13 @@ export default class MicroBitBoard extends EventEmitter {
    * @throws {Error} if called before connecting to firmware
    */
   initializeComponents() {
+    this.boardClientWithoutPort = Object.assign({}, this.boardClient_);
```

Comment:
Could you add a comment here explaining why this is needed?

Reply:
Sure!